web/server/codechecker_server/server.py

```diff
@@ -237,11 +239,14 @@ def do_GET(self):
         # Check that path contains a product endpoint.
         if product_endpoint is not None and product_endpoint != '':
             self.path = self.path.replace(f"{product_endpoint}/", "", 1)
+            # Remove extra leading slashes, see cpython#93789.
+            self.path = '/' + self.path.lstrip('/')
 
         if self.path == '/':
             self.path = "index.html"
 
         # Check that the given path is a file.
+        # The base directory is set to www_root.
         if not os.path.exists(self.translate_path(self.path)):
```

Comment:
You already deleted this translate_path fuction so I think you would have modified this if state.

Reply:
I deleted the re-implementation of the translate_path function, this calls SimpleHTTPRequestHandler.translate_path() - translating the path with the root set to www_root.